janitor/__init__.py

```diff
@@ -1,6 +1,16 @@
 from .functions import *  # noqa: F403, F401
+from .ml import get_features_targets as _get_features_targets
+from .utils import refactored_function
 
 # from .dataframe import JanitorDataFrame as DataFrame  # noqa: F401
 # from .dataframe import JanitorSeries as Series  # noqa: F401
 
+
+@refactored_function(
```

Comment:
❤️ this! Very thoughtful of you, I like this pattern, and I will definitely start doing this in the future!

Reply:
👍 I found this solution online and linked it in `refactored_function`!